lib/sharezone_widgets/lib/src/widgets.dart

```diff
@@ -171,8 +173,8 @@ class DeleteButtonWithPoppingTrue extends StatelessWidget {
 
 enum LogoColor {
   white,
-  blue_long,
-  blue_short,
+  blueLong,
+  blueShort,
 }
 
```

Comment:
@nilsreichardt Yes since I had to change this enum because of a lint, I needed to update its usages inside `app` as well.

Reply:
makes sense haha